packages/app/tailwind.config.js

```diff
@@ -3,20 +3,24 @@ module.exports = {
   content: [
     "./app/**/*.{js,ts,jsx,tsx,mdx}",
     "./ui/**/*.{js,ts,jsx,tsx,mdx}",
-    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}"
   ],
   theme: {
     fontFamily: {
-      sans: ["system-ui", "sans-serif"], // will be Stabil Grotesk
+      sans: ["system-ui", "sans-serif"] // will be Stabil Grotesk
     },
     boxShadow: {
       xs: " 0px 1px 2px -1px rgba(17, 12, 34, 0.08)",
       sm: "0px 2px 4px -2px rgba(17, 12, 34, 0.12)",
       md: "0px 6px 16px -6px rgba(17, 12, 34, 0.1)",
       lg: "0px 16px 20px -8px rgba(17, 12, 34, 0.1)",
       xl: "0px 20px 24px -10px rgba(17, 12, 34, 0.1)",
-      "2xl": "0px 32px 32px -12px rgba(17, 12, 34, 0.12)",
+      "2xl": "0px 32px 32px -12px rgba(17, 12, 34, 0.12)"
     },
+    /**
+     * @TODO rename the palette, use more descriptive color names,
+     * keep the gradients on multiples of 50
+     */
```

Comment:
We'll probably need an improvement on this. 
We'll do it with Akash so we can have the same names on design and code.

Reply:
Yeah, totally agree. In my experience, it's easier for both sides to agree on a naming convention for the color palette around `colorTone[shadeNumber]`

Something like 
`green[200]`

Shade numbers going on multiples of 50 also proved to be quite effective but I'm 100% open to suggestions and discussion :D

Reply:
Let's see that after the MVP. I think will have to merge Akash design conventions with Tailwind. They do the 100, 200, ..., 900 scale. But open to explore other paths.

Reply:
I would remove this comment tho.